lib/ansible/plugins/lookup/skydive.py

```diff
@@ -36,14 +36,14 @@
       description: a dict object that is used to filter the return objects
 """
 
-EXAMPLES = """
+EXAMPLES = r"""
 - name: return skydive metdata if present based on Name
   set_fact:
-    skydive_meta: "{{ lookup('skydive_lookup', filter={'query': \"G.V().Has('Name', 'sumit-VirtualBox')\"}) }}"
+    skydive_meta: "{{ lookup('skydive', filter={'query': \"G.V().Has('Name', 'sumit-VirtualBox')\"}) }}"
 
 - name: return skydive metdata if present based on TID
   set_fact:
-    skydive: "{{ lookup('skydive_lookup', filter={'query': \"G.V().Has('Name', 'sumit-VirtualBox')\"},
+    skydive: "{{ lookup('skydive', filter={'query': \"G.V().Has('Name')\"},
```

Comment:
The query should be something like 

```
G.V().Has('TID', 'bd7f5334-115c-5943-487d-a77c486fa854')
```

according to the `name`

Reply:
updated the `name` field.